### DISCUSSION TITLE
Running on older browser versions

### DISCUSSION THREAD

**Discussion by keponk** [cypress-io/cypress#14247] [Questions and Help]
I'm looking into cypress as a possible candidate to test react apps that will run on an embedded device. This device will be running chromium v51 ~2016. I don't have relationship to the device itself to bump up the chromium version so that's what I get to deal with.

I was wondering a couple things
1) is it at all possible to configure cypress to use a chromium v51? 
2) if possible, would this be recommended?

I could of course wave away the browser version and trust my webpack config to spit out ES5 javascript, but I would feel better to know I can test against the specific browser I care about. I want to leverage the fact that I know exactly which browser/version my users will be running this app on.

**Comment by jennifer-shehane** [comment]
You can download an earlier Chromium version and run Cypress within that downloaded browser. We have a utility to help find older versions of Chromium https://chromium.cypress.io/ but our utility only goes back so far, not to 51. 

So yes, you can download Chromium 51 and run Cypress against it. We do not guarantee that older versions work as intended since we only test the latest version against Cypress.
